locale/moretrees.fr.tr

```diff
@@ -209,3 +209,12 @@ Willow Tree Trunk=Tronc d'arbre de saule
 Willow Tree Trunk Slab=Dalle en tronc de saule
 Willow Tree Trunk Stair=Escalier en tronc de saule
 Yellow=jaune
+
+# init.lua
+Sapling=
```

Comment:
Jeune Arbre / Pousse d'arbre
Arbre
Planches( de bois)
Feuilles
Barrière en bois
Clôture en bois
Porte de clôture en bois

Reply:
'Planches' doesn't need the 'de bois' part when not specifying what type of wood. It's not wrong to add it. 'Planches' are generally made of wood.